src/pages/studyView/StudyViewPage.tsx

```diff
@@ -148,6 +168,14 @@ export default class StudyViewPage extends React.Component<IStudyViewPageProps,
                 props.onResetSelection = this.handlers.resetGeneFilter;
                 props.selectedGenes=this.store.selectedGenes;
                 props.onGeneSelect=this.store.onCheckGene;
+                props.disableGraphicsDownload = true;
+                props.download = [
+                    {
+                        // TODO implement a proper data download function
+                        initDownload: () => Promise.resolve("NA"),
+                        type: 'TSV'
```

Comment:
Let's make the type as a Enum

Reply:
There is a type for this:
```type ChartDownloadType = 'TSV' | 'SVG' | 'PDF';```

This should be enough for now.